packages/bnfgen/bnfgen.2.0/opam

```diff
@@ -0,0 +1,31 @@
+opam-version: "2.0"
+synopsis: "Random text generator that takes context-free grammars from BNF files"
+description: """
+BNFGen generates random texts based on user-defined context-free grammars
+specified in a BNF-like syntax.
+You can specify "weight" for rules with alternation to change the probabilities,
+for example, in `<foo> ::= 10 <foo> "foo" | "foo";` the recursive option will be
+ten times more likely to be taken.
+"""
+maintainer: "Daniil Baturin <daniil@baturin.org>"
+authors: "Daniil Baturin <daniil@baturin.org>"
+license: "MIT"
+homepage: "https://github.com/dmbaturin/bnfgen"
+bug-reports: "https://github.com/dmbaturin/bnfgen/issues"
+dev-repo: "git+https://github.com/dmbaturin/bnfgen"
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name]
+]
+depends: [
+  "ocamlfind" {build}
```

Comment:
Is ocamlfind really still used in the build?

Reply:
Good point. I've updated the PR.